.github/workflows/create-release.yml

```diff
@@ -3,6 +3,7 @@ on:
     workflows: ["CI"]
     types:
       - completed
+  merge_group:
```

Comment:
Is this necessary? Doesn't this mean it essentially runs on PRs?

Reply:
removed!